features/src/llvm/install.sh

```diff
@@ -22,26 +22,55 @@ check_packages                  \
     software-properties-common  \
     ;
 
-if [[ -z "$LLVM_VERSION" \
-      || "$LLVM_VERSION" == "latest" \
-      || "$LLVM_VERSION" == "dev" \
-      || "$LLVM_VERSION" == "pre" \
-      || "$LLVM_VERSION" == "prerelease" \
-]]; then
-    LLVM_VERSION="latest";
-    find_version_from_git_tags \
-        LLVM_VERSION \
-        https://github.com/llvm/llvm-project \
-        "tags/llvmorg-" "." "-init" "true";
-    LLVM_VERSION="$(echo $LLVM_VERSION | grep -oP '[0-9]+')";
-fi
+install_clang_grace() {
+    if [[ "$(uname -p)" != "aarch64" ]]; then
+        echo "Clang for Grace is only supported on aarch64"
+        exit 0;
+    fi
+
+    echo "Installing NVIDIA Clang for Grace"
+    wget -O clang-grace-toolchain.deb https://developer.nvidia.com/downloads/assets/grace/clang/16.23.08/clang-grace-toolchain_16.23.08_arm64.deb
```

Comment:
Question: Are we okay hard coding the version for now?

Reply:
I didn't see another version publicly. We can devise a mechanism later probably.